src/libsyntax/print/pprust.rs

```diff
@@ -714,6 +714,22 @@ pub trait PrintState<'a> {
         Ok(())
     }
 
+    fn print_attribute_path(&mut self, path: &ast::Path) -> io::Result<()> {
```

Comment:
This is a copy of `State::print_path`, like already mentioned here [comment](https://github.com/rust-lang/rust/pull/47773/files#r164904010).

Should I move the `print_path` method to the `PrintState` trait? Or use `self.writer().word(&path_to_string(path))`?